src/patternfly/components/Menu/menu.scss

```diff
@@ -0,0 +1,294 @@
+  --pf-c-menu__item-action--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-action--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-menu__item-action-icon--Color: var(--pf-global--Color--dark-200);
+  --pf-c-menu__item-action-icon--Height: calc(var(--pf-c-menu__item--FontSize) * var(--pf-c-menu__item--LineHeight));
+  --pf-c-menu__item-action--hover__icon--Color: var(--pf-global--Color--dark-100);
+  --pf-c-menu__item-action--m-favorite__icon--Color: var(--pf-global--disabled-color--200);
+  --pf-c-menu__item-action--m-favorite__icon--FontSize: var(--pf-global--icon--FontSize--sm);
+  --pf-c-menu__item-action--m-favorite--m-favorited__icon--Color: var(--pf-global--palette--gold-400);
+
+  padding-top: var(--pf-c-menu--PaddingTop);
+  padding-bottom: var(--pf-c-menu--PaddingBottom);
+  background-color: var(--pf-c-menu--BackgroundColor);
+  box-shadow: var(--pf-c-menu--BoxShadow);
+
+  &.pf-m-flyout {
+    .pf-c-menu__sublist {
+      position: absolute;
+      top: var(--pf-c-menu--m-flyout__sublist--Top);
+      left: 100%;
+    }
+
+    .pf-c-menu__list-item {
+      &:hover,
+      &:focus-within {
+        color: unset;
+        background-color: unset;
+      }
+    }
+
+    .pf-c-menu__item {
+      &:hover,
+      &:focus-within {
+        color: var(--pf-c-menu__list-item--hover--Color);
+        background-color: var(--pf-c-menu__list-item--hover--BackgroundColor);
+      }
+    }
+  }
+
+  .pf-c-divider {
+    margin-top: var(--pf-c-menu--c-divider--MarginTop);
+    margin-bottom: var(--pf-c-menu--c-divider--MarginBottom);
+  }
+}
+
+// Header
+.pf-c-menu__header {
+  padding-top: var(--pf-c-menu__header--PaddingTop);
+  padding-right: var(--pf-c-menu__header--PaddingRight);
+  padding-bottom: var(--pf-c-menu__header--PaddingBottom);
+  padding-left: var(--pf-c-menu__header--PaddingLeft);
+
+  .pf-c-menu__item {
+    --pf-c-menu__item--PaddingTop: 0;
+    --pf-c-menu__item--PaddingRight: 0;
+    --pf-c-menu__item--PaddingBottom: 0;
+    --pf-c-menu__item--PaddingLeft: 0;
+  }
+}
+
+.pf-c-menu__sublist {
+  padding-top: var(--pf-c-menu__sublist--PaddingTop);
+  padding-bottom: var(--pf-c-menu__sublist--PaddingBottom);
+  background-color: var(--pf-c-menu__sublist--BackgroundColor);
+  box-shadow: var(--pf-c-menu__sublist--BoxShadow);
+}
+
+// List item
+.pf-c-menu__list-item {
+  position: relative;
+  display: flex;
+  color: var(--pf-c-menu__list-item--Color);
+  background-color: var(--pf-c-menu__list-item--BackgroundColor);
+
+  &:hover:not(.pf-m-disabled),
+  &:focus-within:not(.pf-m-disabled) {
+    --pf-c-menu__list-item--Color: var(--pf-c-menu__list-item--hover--Color);
+    --pf-c-menu__list-item--BackgroundColor: var(--pf-c-menu__list-item--hover--BackgroundColor);
+
+    > .pf-c-menu__sublist > .pf-c-menu__list-item {
+      --pf-c-menu__list-item--Color: unset;
+      --pf-c-menu__list-item--BackgroundColor: none;
+    }
+
+    .pf-c-menu__item-external-icon {
```

Comment:
you'll want to scope this class in case there are nested links in submenus

Reply:
do you think it should go inside of `pf-c-menu__item` or `pf-c-menu__item-main` ?